notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb

```diff
@@ -33,18 +33,18 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    delete_bucket = False
</pre><p>We actually want to delete buckets for testing or else they pile up. Use the envvar to check.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/649/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>